frontend/ui/translator.lua

```diff
@@ -220,6 +220,10 @@ function Translator:genSettingsMenu()
         sub_item_table = {
             {
                 text = _("Auto-detect source language"),
+                help_text = _("This setting is best suited for foreign sentences or text fragments found in books written in your native language."),
```

Comment:
A bit roundabout imo.
```suggestion
                help_text = _("This setting is best suited for foreign text found in books written in your native language."),
```